### PR TITLE
vlc: add page

### PR DESCRIPTION
**Pull request by Rudloff**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by zdroid** [comment]
I'm suprised this command hasn't been added yet.

**Comment by Rudloff** [comment]
My thoughts exactly :slightly_smiling_face:

File: pages/common/vlc.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+- Play an audio or video file:
+
+`vlc {{path/to/file}}`
+
+- Play in fullscreen:
+
+`vlc --fullscreen {{path/to/file}}`
+
+- Mute the audio:
```
Comment (andrik) [inline pages/common/vlc.md:15]:
```suggestion
- Execute and mute a multimedia file:
```

File: pages/common/vlc.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# vlc
+
+> VLC media player.
+> Cross-platform multimedia player.
+> More information: <https://www.videolan.org/vlc/>.
+
+- Play an audio or video file:
```
Comment (andrik) [inline pages/common/vlc.md:7]:
```suggestion
- Execute a multimedia file:
```

File: pages/common/vlc.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+`vlc {{path/to/file}}`
+
+- Play in fullscreen:
```
[the remark this answers is not in the document]
Comment (fejx) [reply pages/common/vlc.md:11]: In my opinion, "play" is more concise and just as expressive. I would expect your phrasing more in a proper manpage, but tldr is meant to be compact and easy to read.

File: pages/common/vlc.md (hunk 1 above)
Comment (fejx) [reply pages/common/vlc.md:15]:
```suggestion
- Play muted:
```

File: pages/common/vlc.md (hunk 2 above)
Comment (fejx) [reply pages/common/vlc.md:7]:
```suggestion
- Play a file:
```

File: pages/common/vlc.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Rudloff) [reply pages/common/vlc.md:11]: I feel like "execute" is a bit overly technical here.

File: pages/common/vlc.md (hunk 2 above)
Comment (Rudloff) [reply pages/common/vlc.md:6]: Maybe?
```suggestion
- Play a multimedia file:
```

Comment (fejx) [reply pages/common/vlc.md:6]: Sure! But isn't it clear that a file that is played by a media player should be a multimedia file?
However, I feel like, the change is not significant enough to be worth arguing about 😄.

File: pages/common/vlc.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (andrik) [reply pages/common/vlc.md:10]: :+1:

**Pull request merged**